R/calculations.R

```diff
@@ -256,3 +256,93 @@ hd_int <- function(x, level = 0.95) {
   attr(hdinterval, "conf.level") <- level
   hdinterval
 }
+
+
+#' Effect sizes for Analysis of Variance
+#'
+#' Calculates effect-size measures for Analysis of Variance output objects.
+#'
+#' @param x An object of class \code{apa_variance_table}.
+#' @param es Character. A vector naming all to-be-computed effect-size measures.
+#'   Currently, partial eta-quared (\code{"pes"}), generalized eta-squared
+#'   (\code{"ges"}), and eta-squared (\code{"es"}) are supported.
+#' @param observed Character. A vector naming all factors that are observed
+#'   (i.e., \emph{not} manipulated).
+#' @param mse Logical. Should means-squared errors be computed?
+#' @param intercept Logical. Should the sum of squares of the intercept (i.e., the
+#'   deviation of the grand mean from 0) be included in the calculation of eta-squared?
```

Comment:
Maybe we should add something like "Set to true if a deviation of the grand mean from zero constitutes to-be-explained variance."?

Reply:
Hmm, dunno. I think this sounds a bit too much like a well-justified advice in statistical reasoning, something I don't claim for my within-package documentation.

Reply:
Fair enough. :)